ecs/sdk.go

```diff
@@ -565,7 +565,7 @@ func (s sdk) ListStackServices(ctx context.Context, stack string) ([]string, err
 			return nil, err
 		}
 		for _, r := range response.StackResourceSummaries {
-			if aws.StringValue(r.ResourceType) == "AWS::ECS::Service" {
+			if aws.StringValue(r.ResourceType) == "AWS::ECS::Services" {
```

Comment:
is this a bug fix (unrelated to events) ?

Reply:
oups